### PR TITLE
T/8 UI fixes.

### LINKED ISSUE #8: UI issues
That's how the page should look like:

docs_pt-sans

This is how it looks like now:

screen shot 2016-09-12 at 16 35 35

Issues which I noticed:

* Wrong background color for items in the left menu.
* Wrong font color (everywhere).
* Wrong font size in the ToC.
* Wrong font-weight in the top menu.
* Too big "CKEditor 5" in the top bar.
* Doubled border on the left side of ToC.
* Too big margin on the left side of the ToC.
* Wrong margins between items in the ToC.
* Too dark arrow icons.
* Far too high items in the left menu.
* Code samples were meant to have dark background.

Other issues are related to RWD:

* The middle column was supposed to scale together with the window up to some max width (which makes sense for it. See how it works on http://docs.ckeditor.com now.
* No version for small screens.

### PR DESCRIPTION
**Pull request by postJScriptum**
Fixes #8. 

Fixes also #24 and removed breadcrumbs part from #23 (parent category is still missing).

### CODE REVIEW

File: assets/scripts/script.js [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import * as disqus from './modules/disqus';
 import * as jsfiddle from './modules/jsfiddle';
 import * as dropdown from './modules/dropdown';
+import * as cotController from './modules/cotController';
```
Comment (Reinmar) [inline assets/scripts/script.js]: We use only lower case letters in file names. So it should be `toc-controller` or simply `toc`. And not COT of course :)

File: assets/styles/style.scss [hunk 2]
```diff
@@ -23,17 +31,42 @@ pre code {
 	height: 100%;
 }
 
+.section {
+	display: inline-block;
+	width: 100%;
+
+	&--flex {
```
Comment (Reinmar) [inline assets/styles/style.scss]: Can we use flex if we're going to support (perhaps) IE11 one day?

**Comment by Reinmar** [comment]
screen shot 2016-09-16 at 13 59 28

OK, it doesn't look good unfortunately. Either we need to align the whole thing (3 cols) to the center when the screen gets so big or we need to at least center the middle column like on http://redux.js.org/.

**Comment by Reinmar** [comment]
Other things that I noticed:

1. Make the ToC wider, cause even slightly longer content won't fit there.
2. Fix the double scroll when the viewport gets narrow (or we can postpone it to #20 - it's up to you)
3. `.menu-dropdown__label` positions shouldn't have `cursor: pointer`. The same problem with the middle level positions – they are not links but they have the wrong cursor.
4. I'd consider a background change (without a text-underline) for hovers in the left menu and top menu (can be done later – then please report a "UI polishing" ticket).
5. Font size in the ToC is 13.sth px, while in the menu on the left it's round 14px.

**Comment by Reinmar** [comment]
The double scroll issue:

**Comment by Reinmar** [comment]
To the "UI polishing" ticket you can also add the thing with incorrect position of the scroll bars in the code samples.

File: assets/styles/style.scss (hunk 2 above)
Comment (postJScriptum) [inline assets/styles/style.scss]: Unfortunately we can't, so I've changed to regular display block.

**Comment by Reinmar** [comment]
I noticed two things:

1. The borders of the left menu and ToC are gone now.
2. The hover style in the menu doesn't really work visually. The background would need to be stretched to the whole width of the menu, but I don't think that this would good... So let's use the same styling as in the ToC.
3. There's "active" class available so we should style it. For KISS, we can use the hover style.


screen shot 2016-09-19 at 10 17 50
screen shot 2016-09-19 at 10 17 56

**Comment by Reinmar** [comment]
PS. I know that the `active` class is assigned to every item... This is fault of the too simple test files which reuse the same guide everywhere. Don't worry with that.

**Comment by pomek** [comment]
I will do something with the duplicate classes. Maybe I should add more "example" files to avoid complications.

**Comment by pomek** [comment]
I've reported an [issue ](https://github.com/cksource/docs-builder/issues/40) and I'm working on that.

**Comment by postJScriptum** [comment]
Borders to side navigation and ToC has been added. Active link (and hovered) on side navigation has the same style as on ToC.

**Comment by Reinmar** [comment]
The borders weren't added correctly:

screen shot 2016-09-19 at 22 24 54
screen shot 2016-09-19 at 22 24 58

And the styling on the left isn't the same as on the right. On the right there's underline on hover, while on the left there's only color.

My proposal:

1. Let active be marked only with color.
2. Let hover mean color+underline.

**Comment by Reinmar** [comment]
There are still two issues, both visible on the below screenshot:

screen shot 2016-09-20 at 14 16 25

The ToC's border is too short. In fact.... I just noticed that the same problem is with the left menu :<.

Additionally, the icons (menu and toc) are not aligned vertically.

**Pull request merged**